infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cn.hippo4j.common.toolkit.logtracing;
+
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class LogMessageTest {
+
+    private final static String MESSAGE = "message";
+    private final static String THROWABLE_MESSAGE = "throwable message";
+
+    private LogMessage logMessage;
+
+    @BeforeEach
+    public void init() {
+        logMessage = LogMessage.getInstance();
+    }
+
+    @Test
+    void getInstance_shouldReturnANewLogMessageInstance() {
```

Comment:
Methods should be camelCased

Reply:
The method name should start with test, plus the specific method name

Reply:
https://github.com/opengoofy/hippo4j/pull/1340

Reply:
Thanks, I will follow the project conventions, so will change it in a bit :-)

But I just want to explain what I think about it: 
_Unit test methods don't need to be called by our code or represents actual requests. They have the freedom to tell a story, and spacing between words helps with that. While spaces aren't allowed in method names, using underscores comes close._

I mean for me would be even better to write the **test** method in this way:
```
void getInstance_should_return_a_new_log_message_instance()
 ```

Reply:
From the point of view of Java method naming, I recommend you to use camel case naming.

Reply:
just pushed thanks :-)